CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
   optional, and signon itself manages this during oauth handshake (see:
   [RFC 78](https://github.com/alphagov/govuk-rfcs/blob/8cbb2a0de86de02f54ae37b245e79b46ad62cb6a/rfc-078-re-architect-signin-permissions-in-signon.md))
 
+# 13.2.1
+
+* README fix
```

Comment:
I don't think this is relevant in the changelog for a gem, the above bit around deprecating a method is though, so I'd suggest just changing "Development" to "13.3.0" (as this is a minor backwards compatible change)?